src/table.ts

```diff
@@ -1775,7 +1775,7 @@ class Table extends common.ServiceObject {
     });
 
     if (!options.raw) {
-      json.rows = arrify(rows).map((row) => {
+      json.rows = rows.map((row: RowMetadata) => {
```

Comment:
If `rows` is typed, why do you need to be explicit about the type here?

Reply:
Interestingly this throws an implicit any, I'm guessing its because `RowMetadata` is actually just an alias to `any`.